initfiles/etc/DIR_NAME/environment.conf.in

```diff
@@ -12,6 +12,7 @@ pid=${PID_PATH}
 log=${LOG_PATH}
 user=${RUNTIME_USER}
 group=${RUNTIME_GROUP}
+#umask=022
```

Comment:
I added it back in and left it as a comment as I think it helps document it as an option.